README.rst

```diff
@@ -30,7 +30,21 @@ This is easily achieved by downloading
 Usage Example
 =============
 
-.. todo:: Add a quick, simple example. It and other examples should live in the examples folder and be included in docs/examples.rst.
+.. code-block:: python
+
+	import time
+	import board
+	import busio
+	import adafruit_adt7410
+
+	i2c_bus = busio.I2C(board.SCL, board.SDA)
+	adt = adafruit_adt7410.ADT7410(i2c_bus, address=0x48)
+	adt.high_resolution = True
+
+	while True:
+		print(adt.temperature)
```

Comment:
Might want something similar to the MCP9808 temperature example to show F and C readings:
```
    tempC = adt.temperature
    tempF = tempC * 9 / 5 + 32
    print('Temperature: {} C {} F '.format(tempC, tempF))
    time.sleep(0.5)
```

Reply:
i like to keep it simple 😁